pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -684,3 +752,45 @@ def test_link_object_of_complex_object_should_also_be_protected_from_deletion(
                     self.shell,
                     self.cluster.default_rpc_endpoint,
                 )
+
+    @allure.title("Locked object should be protected from deletion after the storage nodes are restarted")
+    @pytest.mark.parametrize(
+        # Only complex objects are required for this test
+        "new_locked_storage_object",
+        [pytest.lazy_fixture("complex_object_size")],
+        indirect=True,
+    )
+    def test_the_object_lock_should_be_kept_after_restarting_the_nodes(
+            self,
+            new_locked_storage_object: StorageObjectInfo,
+            enable_metabase_resync_on_start,
+    ):
+        """
+        Lock objects should fill metabase on resync_metabase
+        """
+        with allure.step(f"Delete metabase files from storage nodes"):
+            for node in self.cluster.storage_nodes:
+                delete_node_data(node)
```

Comment:
But does it remove metabase only? Looks like it drops everything and that's a radically different thing. The node doesn't have any data at all in this case and there is nothing to resync the metabase from.

Reply:
> But does it remove metabase only? Looks like it drops everything and that's a radically different thing. The node doesn't have any data at all in this case and there is nothing to resync the metabase from.

Fixed